src/stylesheets/components/_header.scss

```diff
@@ -142,83 +138,83 @@ $z-index-nav:     9000;
 
 .usa-header-extended {
   .usa-header {
-    @include media($nav-width) {
+    @include at-media($theme-nav-width) {
       border-bottom: none;
     }
   }
 
   .usa-logo {
-    @include media($nav-width) {
-      @include u-margin-y(3rem);
+    @include at-media($theme-nav-width) {
+      @include u-margin-y(4);
     }
   }
 
   em {
-    @include media($nav-width) {
-      font-size: 3.2rem;
-      line-height: 1;
+    @include at-media($theme-nav-width) {
+      font-size: scale($theme-font-family-header, 'xl');
+      line-height: line-height($theme-font-family-header, 1);
     }
   }
 
   .usa-navbar {
-    @include media($nav-width) {
-      @include outer-container($site-max-width);
-      @include u-padding-x($site-margins);
+    @include at-media($theme-nav-width) {
+      @include outer-container($site-max-width); // TODO: update
```

Comment:
Tried to use `@include grid-container` instead of this, but it caused issues that I couldn't identify, cc @thisisdano.

Reply:
I'll try to replicate and see what's up

Reply:
That's unusual. It's not just `grid-container` that triggers this behavior, but using neither and just adding the padding and max width manually also results in wonky output.

Reply:
Adding `overflow: auto;` helps — is there some float clearing thing happening?

Reply:
there's a clearfix on this https://github.com/uswds/uswds/pull/2746/files#diff-70c90cf98b07e2f38faec5eb1e9fa4e3R188

Reply:
Done in 30937b3. Had to add `!important` to change the grid's padding tho.